src/cloudHandler.js

```diff
@@ -11,7 +14,11 @@ var handler = function (params, success, failure) {
         cloudPath = '/sync/';
     }
     var url = cloudURL + cloudPath + params.dataset_id;
-    var json = JSON.stringify(params.req);
+    var payload = params.req;
+    payload.__fh = {
```

Comment:
I know this is from the old js sdk, but I am wondering if this is the place. Given that the sync server requires the client to send a unique device id in the payload, should this logic be moved to the the place where the pay load is built - the end results is the same, but it will keep them in the same place, and people don't have to look at 2 places to figure out how the payload is built.

Also would really like to see `__fh` go way....